package.json

```diff
@@ -24,6 +24,7 @@
   "author": "Ocean Protocol",
   "license": "Apache-2.0",
   "scripts": {
+    "release": "release-it --non-interactive",
```

Comment:
`--non-interactive` isn't doing anything anymore and we actually want the interactive mode https://github.com/release-it/release-it#interactive-vs-ci-mode so can be removed